reflex/components/datadisplay/code.py

```diff
@@ -411,6 +411,12 @@ class CodeBlock(Component):
     # Props passed down to the code tag.
     code_tag_props: Var[Dict[str, str]]
 
+    # Whether a copy button should appear.
+    can_copy: Var[Optional[bool]] = Var.create(False)
```

Comment:
The other problem here is now these are annotated and treated as `Var`... but in many cases, the code in the `create` is using python `if` statements and potential other operations that are not valid on `Var` types.

I'd be more okay defining these as regular model fields (i.e. without the `Var`), but then i don't think they'd be automatically picked up by the pyi generator and the doc generation

Reply:
The regular model fields get picked up by the pyi generator and reflex-web. Some kwargs (that were moved) accept literals or vars. Maybe in this case it might be worth specifying the typing of these model fields to accept both literals and vars